src/components/forms/shipment/ShipmentDetailsData.tsx

```diff
@@ -0,0 +1,151 @@
+              <Select<Object, true, GroupBase<Object>>
+                name={name}
+                ref={ref}
+                value={value}
+                onChange={onChange}
+                options={deliveryStatusOptions}
+                placeholder={format('shipment.delivery_status.label')}
+              />
+            )}
+            name="delivery_status"
+          />
+        </div>
+        <div className="flex flex-col gap-2 w-1/3">
+          <p className="text-xs text-transit-black-secondary font-medium">{format('shipment.pod_status.label')}</p>
+          <Controller
+            rules={{ required: true }}
+            control={control}
+            render={({
+              field: {
+                onChange, value, name, ref,
+              },
+            }) => (
+              <Select<Object, true, GroupBase<Object>>
+                name={name}
+                ref={ref}
+                value={value}
+                onChange={onChange}
+                // eslint-disable-next-line
+                options={watch('delivery_status') !== null && watch('delivery_status') !== undefined ? (watch('delivery_status').value === 'delivered' ? podStatusDeliveredOptions : podStatusNotDeliveredOptions) : []}
+                placeholder={format('shipment.pod_status.label')}
+              />
+            )}
+            name="pod_status"
+          />
+        </div>
+        <div className="flex flex-col gap-2 w-1/3">
+          <p className="text-xs text-transit-black-secondary font-medium">{format('shipment.signed_by.label')}</p>
+          <Input
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...register('signed_by')}
+            name="signed_by"
+            id="floatingInput"
+            placeholder={format('shipment.signed_by.label')}
+            type="text"
+            className="h-9 placeholder-grey-300"
+          />
+        </div>
+      </div>
+      <div className="flex flex-row gap-2">
+        <div className="flex flex-col gap-2 w-1/3">
+          <p className="text-xs text-transit-black-secondary font-medium">{format('shipment.delivery_date.label')}</p>
+          <Controller
+            control={control}
+            render={({ field: { onChange, value } }) => (
+              <DatePick
+                currentDate={value}
+                onChange={onChange}
+              />
+            )}
+            name="delivery_date"
+          />
+        </div>
+        <div className="flex flex-col gap-2 w-1/3">
+          <p className="text-xs text-transit-black-secondary font-medium">{format('shipment.gps_coordinates.label')}</p>
+          <Input
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...register('gps_coordinates')}
+            name="gps_coordinates"
+            id="floatingInput"
+            placeholder={format('shipment.gps_coordinates.label')}
+            type="text"
+            className="h-9 placeholder-grey-300"
```

Comment:
`placeholder-grey-300` not necessary

Reply:
ok. I will remove this later while working on another tickets.